mappings/net/minecraft/world/World.mapping

```diff
@@ -344,15 +404,20 @@ CLASS net/minecraft/class_99 net/minecraft/world/World
 	METHOD method_394 spawnEntity (Lnet/minecraft/class_1745;)Z
 		ARG 1 entity
 	METHOD method_396 doesAreaContainFireSource (Lnet/minecraft/class_646;)Z
-		ARG 1 boundingbox
+		ARG 1 area
```

Comment:
`Box` as `area` sounds conc

Reply:
What I take more issue in is the long name `doesXContainY` could just be `containsY`.

Reply:
I renamed it to so it's consistent with the others. What wrong about `area`?

Reply:
`Box` can be named as `box`. self explanatory

Reply:
or `boundingBox`

Reply:
please not `area`

Reply:
`area` is also a 2 dimensional term
gets confusing

Reply:
> `area` is also a 2 dimensional term
> gets confusing

`volume` would be the correct alternative then.

Reply:
> > `area` is also a 2 dimensional term
> > gets confusing
> 
> `volume` would be the correct alternative then.

doesn't matter

Reply:
upstream uses box

Reply:
`public <T extends Entity> List<T> getEntitiesByType(@Nullable EntityType<T> type, Box box, Predicate<? super T> predicate)`  
this has box

Reply:
fixed

Reply:
still says area here 🤔